src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -415,75 +423,75 @@ group target (CN is a reasonable default)
      * @deprecated retained for backwards binary compatibility.
      */
     @Deprecated
-    public LDAPSecurityRealm(String server, String rootDN, String userSearchBase, String userSearch, String groupSearchBase, String managerDN, String managerPassword, boolean inhibitInferRootDN) {
-        this(server, rootDN, userSearchBase, userSearch, groupSearchBase, managerDN, managerPassword, inhibitInferRootDN, false);
+    public LDAPSecurityRealm(String server, boolean sslVerify, String rootDN, String userSearchBase, String userSearch, String groupSearchBase, String managerDN, String managerPassword, boolean inhibitInferRootDN) {
```

Comment:
these are old deprecated constructors and are kept here to keep backwards binary compatibility. So there signature should never change.

Reply:
New commit resolved. Please help review.